web/public/js/txadmin/players.js

```diff
@@ -45,462 +50,314 @@ plistSearchElement.addEventListener('input', function (ev) {
 // });
 
 //Handle Remove, Add and Update playerlist
-function removePlayer(player){
-    document.getElementById(`divPlayer${player.id}`).remove();
+function removePlayer(player) {
+  document.getElementById(`divPlayer${player.id}`).remove();
 }
 
-function addPlayer(player){
-    let div = `<div class="list-group-item list-group-item-accent-secondary player text-truncate" 
+function addPlayer(player) {
+  let div = `<div class="list-group-item list-group-item-accent-secondary player text-truncate" 
                 onclick="showPlayer('${player.license}')" id="divPlayer${player.id}">
                     <span class="pping text-secondary">&nbsp;??</span>
                     <span class="pname">#${player.id}</span>
-            </div>`
-    $("#playerlist").append(div);
+            </div>`;
+  $("#playerlist").append(div);
 }
 
-function updatePlayer(player){
-    let el = document.getElementById(`divPlayer${player.id}`);
-
-    let pingClass;
-    player.ping = parseInt(player.ping);
-    if (player.ping < 0) {
-        pingClass = 'secondary';
-        player.ping = '??';
-    } else if (player.ping < 60) {
-        pingClass = 'success';
-    } else if (player.ping < 100) {
-        pingClass = 'warning';
-    } else {
-        pingClass = 'danger';
-    }
-
-    el.classList.remove('list-group-item-accent-secondary', 'list-group-item-accent-success', 'list-group-item-accent-warning', 'list-group-item-accent-danger');
-    el.classList.add('list-group-item-accent-' + pingClass);
-    el.firstElementChild.classList.remove('text-secondary', 'text-success', 'text-warning', 'text-danger');
-    el.firstElementChild.classList.add('text-' + pingClass);
-    el.firstElementChild.innerHTML = player.ping.toString().padStart(3, 'x').replace(/x/g, '&nbsp;');
-    el.lastElementChild.textContent = player.name;
-    el.dataset['pname'] = player.name.toLowerCase();
+function updatePlayer(player) {
+  let el = document.getElementById(`divPlayer${player.id}`);
+
+  let pingClass;
+  player.ping = parseInt(player.ping);
+  if (player.ping < 0) {
```

Comment:
could just check for falsey here, as if `parseInt` returns NaN the condition won't return true as `NaN` isn't less than `0`, nor greater than, nor equal to

but, if you check for falsey then 0 is falsey and so is NaN